imx-bootlets: Fix cmdlines for Linux boot

This adapt the cmdlines for the partitioning layout used on the
generated SD cards.

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Otavio Salvador <otavio@ossystems.com.br>

diff --git a/linux_prep/cmdlines/iMX28_EVK.txt b/linux_prep/cmdlines/iMX28_EVK.txt
index 6bc36b9..6cf89a0 100644
--- a/linux_prep/cmdlines/iMX28_EVK.txt
+++ b/linux_prep/cmdlines/iMX28_EVK.txt
@@ -1 +1 @@
-console=ttyAM0,115200 root=/dev/mmcblk0p3 rw rootwait
+console=ttyAMA0,115200 root=/dev/mmcblk0p2 rw rootwait
diff --git a/linux_prep/cmdlines/stmp378x_dev.txt b/linux_prep/cmdlines/stmp378x_dev.txt
index 9a717eb..fdb7390 100644
--- a/linux_prep/cmdlines/stmp378x_dev.txt
+++ b/linux_prep/cmdlines/stmp378x_dev.txt
@@ -1,6 +1,3 @@
-console=ttyAM0,115200 root=/dev/mmcblk0p3 rw rootwait lcd_panel=lms430 no_console_suspend
-console=ttyAM0,115200 root=/dev/mmcblk0p3 rw rootwait lcd_panel=lms350
-console=ttyAM0,115200 ssp1=spi1 ubi.mtd=2 root=ubi0:rootfs0 rootfstype=ubifs lcd_panel=lms430
-
-
-
+console=ttyAMA0,115200 root=/dev/mmcblk0p2 rw rootwait ssp1=mmc lcd_panel=lms430 no_console_suspend
+console=ttyAMA0,115200 root=/dev/mmcblk0p2 rw rootwait ssp1=mmc lcd_panel=lms350
+console=ttyAMA0,115200 ssp1=spi1 ubi.mtd=2 root=ubi0:rootfs0 rootfstype=ubifs lcd_panel=lms430
